src/modules/dogfact.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+const axios = require('axios')
+const { EmbedBuilder } = require('discord.js')
+
+module.exports = {
+    name: 'dogfact',
+    description: 'Sends random fact about dogs.',
+    async execute(msg) {
+        const {data: {facts}} = await axios.get('https://dog-api.kinduff.com/api/facts')
+        await msg.reply({embeds: [new EmbedBuilder().setTitle("Random dog fact!").setDescription(facts[0]).setColor('Random').setTimestamp()]})
```

Comment:
Can you use constructor rather than setters, to match the way I use Embed `lol.js` and `r6.js`? Also please include `Mover Bot` in the footer.

Reply:
Okay, I'll add it.

Reply:
Added!